ee/codegen/src/project.ts

```diff
@@ -416,12 +416,15 @@ ${errors.slice(0, 3).map((err) => {
       );
     }
 
-    const nodesToGenerate = await Promise.all(
+    const nodesToGenerate = await Promise.allSettled(
```

Comment:
idt we need `allSettled`, just `.all` plus a filtering out `null` should suffice. This allows strict errors to still throw